### DISCUSSION TITLE
Hi there. I think I may have a development

### DISCUSSION THREAD

**Discussion by iOSjNaGeR** [dragokas/hijackthis#167] [General]
- [ ] ![image](https://user-images.githubusercontent.com/95143219/144713975-1c3559ca-12e5-4d8e-86d2-c4273b8436f0.jpeg)Tool that is not exactly legal or safe??!!Used to compose programs years ago & looking to try again. Its changed a lot but the tool I’m using doesn’t seem right. Apologies if I have upset you. Never my intention. Tool I’ve been using is called “working [copy” @]()![image](https://user-images.githubusercontent.com/95143219/144713981-a7431f68-8705-4f3e-be6a-d12a2709ca76.jpeg)[![image](https://user-images.githubusercontent.com/95143219/144713995-08d5cb0c-dd68-415b-a4d1-1d500c03e44f.jpeg)]()

**Comment by iOSjNaGeR** [comment]
I've stopped using it cos it don't feel right

**Comment by dragokas** [comment]
Hi,
Are you asking about possibility to check a specific software for malware behaviour?

**Reply by iOSjNaGeR** [reply]
Sent you email kind lady ☺️

**Reply by iOSjNaGeR** [reply]
Hey there. I am not too worried now if I'm honest. Done some research & I'm running latest  V..iOS so it should be okay. Sorry to have troubled you. Take care ☺️😊

**Comment by iOSjNaGeR** [comment]
Yes that would be good to understand if I am in fact using an illegal piece of software that I downloaded from the Apple store

Sorry if it’s been an inconvenience to you.

Kind Regards,

Jay
________________________________

**Comment by dragokas** [comment]
Is that software for mobile phone?
You can submit suspicious sample to one of AV vendor's form/email mentioned in the topic: https://www.safezone.cc/threads/kak-i-kuda-mozhno-otpravit-podozritelnye-fajly-na-analiz.5/
We are doing analysis for Windows software samples only and at the moment I can't promise how much time this one will last. I think AV Lab will respond to you faster.

**Comment by iOSjNaGeR** [comment]
Thank you for your insight & help

Kind Regards,

Jay
________________________________
